src/main/java/net/dries007/tfc/objects/items/ItemsTFC.java

```diff
@@ -320,6 +323,7 @@ public static void registerVanillaOverrides(RegistryEvent.Register<Item> event)
             new ItemBlockTorch(Blocks.TORCH).setRegistryName("minecraft", "torch"),
             new ItemGlassBottleTFC().setRegistryName(Items.GLASS_BOTTLE.getRegistryName()).setTranslationKey("glassBottle")
         );
+        CapabilityItemHeat.CUSTOM_ITEMS.put(IIngredient.of(Items.EGG), () -> new ItemHeatHandler(null, 1, 480));
```

Comment:
This should be in `CapabilityItemHeat#preInit`, not here.

Reply:
Thanks for providing correct location, wasn't sure at all.